x-pack/plugins/lists/server/services/items/find_list_item.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getEmptySearchListMock } from '../../../common/schemas/elastic_response/search_es_list_schema.mock';
+import { getCallClusterMockMultiTimes } from '../../../common/get_call_cluster.mock';
+import { getFoundListItemSchemaMock } from '../../../common/schemas/response/found_list_item_schema.mock';
+
+import { getFindListItemOptionsMock } from './find_list_item.mock';
+import { findListItem } from './find_list_item';
+
+describe('find_list_item', () => {
+  beforeEach(() => {
```

Comment:
I don't see anything in here that would require `clearAllMocks`; that's typically for persistent mocks that may carry state between tests. Is this a "just in case" thing or was there a particular reason for it?

Reply:
Probably not, I will remove it here if it's an extra. I think it's just copy-pasta mistake.